netbox/wireless/tables/wirelesslink.py

```diff
@@ -36,6 +37,10 @@ class WirelessLinkTable(TenancyColumnsMixin, NetBoxTable):
         verbose_name=_('Interface B'),
         linkify=True
     )
+    length = columns.TemplateColumn(
```

Comment:
We should replace these two TemplateColumns (here and on CableTable) with a new custom column to avoid the duplicated template code.

Reply:
renamed fields so can no longer use the same